.github/workflows/artifacts.yml

```diff
@@ -127,22 +170,35 @@ jobs:
           node-version: '${{ steps.node_version.outputs.NODE_VERSION }}'
 
       - name: Build
+        env:
+          # Needed until hosted macos-11.0 runners become available
+          SDKROOT: /Library/Developer/CommandLineTools/SDKs/MacOSX11.1.sdk
```

Comment:
This is no good because they might update to 11.2. Is it sufficient to use `xcrun -sdk macosx make` below? `xcrun` automatically sets a bunch of environment variables like SDKROOT but I don't know if it'll pick the latest SDK or the host SDK.